relay-server/src/statsd.rs

```diff
@@ -864,7 +863,7 @@ impl CounterMetric for RelayCounters {
             RelayCounters::BufferEnvelopesRead => "buffer.envelopes_read",
             RelayCounters::BufferEnvelopesReturned => "buffer.envelopes_returned",
             RelayCounters::BufferStateTransition => "buffer.state.transition",
-            RelayCounters::BufferEnvelopeStacksPopped => "buffer.envelope_stacks_popped",
+            RelayCounters::BufferProjectKeyPairPopped => "buffer.project_key_pair_popped",
```

Comment:
I wouldn't necessarily rename the metric. It makes it harder to look back at the history of the metric, + we still have conceptual stacks, so `envelope_stacks` is not wrong IMO.